kong/clustering/init.lua

```diff
@@ -53,22 +48,29 @@ function _M:init_cp_worker(plugins_list)
 
   events.init()
 
+  self.wrpc_handler =
+    require("kong.clustering.wrpc_control_plane").new(self.conf, self.cert_digest)
+
```

Comment:
Notice this was at `init` phase, and this change makes it happen at `init_worker`.